requirements/system-requirements.md

```diff
@@ -53,13 +54,19 @@ Please note that we advise you __not to install the SPDocKit Event Collection Se
   * Installation – 100 MB disk space
   * Index – 200GB disk space required to store the index containing all the ULS logs. It is recommended to change the index location to a secondary drive.
 
+## SharePoint Online
 
-### Learn more
-
-[Installation Guide]()
+### Requirements
 
-[How to create Farm Documentation]()
+From the version 7.0, SPDocKit offers support for SharePoint Online. SPDocKit Online can be installed on any workstation or server with __Windows 7 / Windows Server 2008__ or better. SPDocKit Online needs to be installed on a server that is not a part of the SharePoint On-Premises farm.
```

Comment:
From version 7.0, SPDocKit offers support for SharePoint Online. SPDocKit Online can be installed on any workstation or server with __Windows 7 / Windows Server 2008__ or better. SPDocKit Online needs to be installed on a server that is not a part of the SharePoint On-Premises farm. [articles too many]